code/game/gamemodes/miniantags/demons/shadow demon/shadow_demon.dm

```diff
@@ -0,0 +1,73 @@
+/mob/living/simple_animal/demon/shadow
+	name = "shadow demon"
+	desc = "A creature thats barely tangable, you can feel its gaze pierce you."
```

Comment:
```suggestion
	desc = "A creature that's barely tangible, you can feel its gaze pierce you."
```

Reply:
(Not sure about the pierce/piercing, though, piercing sounds more natural to me)

Reply:
If you're using piercing, it should be something like "piercing through you".